### PR TITLE
[plugin.image.qwantz] 1.0.1

### PR DESCRIPTION
**Pull request by macica2**
### Description
Dinosaur Comics is a humorous comic about talking dinosaurs. It was created by Ryan North and can be viewed freely at http://www.qwantz.com. This add-on lets you view the comics from the site. I received permission from Ryan North before creating this add-on.

This PR is a resubmission of https://github.com/xbmc/repo-plugins/pull/2900 since I messed up that commit and the code base was out of date. I did address the feedback given on that PR.
### Checklist:

- [x] My code follows the [add-on rules](http://kodi.wiki/view/Add-on_rules) and [piracy stance](http://kodi.wiki/view/Official:Forum_rules#Piracy_Policy) of this project. 
- [x] I have read the [CONTRIBUTING](https://github.com/xbmc/repo-plugins/blob/master/CONTRIBUTING.md) document
- [x] Each add-on submission should be a single commit with using the following style: [plugin.video.foo] v1.0.0

Additional information :
- Submitting your add-on to this specific branch makes it available to any Kodi version equal or higher than the branch name with the applicable Kodi dependencies limits.
- [add-on development](http://kodi.wiki/view/Add-on_development) wiki page.
- Kodi [pydocs](http://kodi.wiki/view/PyDocs) provide information about the Python API
- [PEP8](https://www.python.org/dev/peps/pep-0008/) codingstyle which is considered best practice but not mandatory.
- This add-on repository has automated code guideline check which could help you improve your coding. You can find the results of these check at [Codacy](https://www.codacy.com/app/Kodi/repo-plugins/dashboard). You can create your own account as well to continuously monitor your python coding before submitting to repo.
- Development questions can be asked in the [add-on development](http://forum.kodi.tv/forumdisplay.php?fid=26) section on the Kodi forum.
- If you see no activity on your PR after a week (so at least one weekend has passed) then please go to the #kodi-dev freenode IRC channel to reach out to the team

### CODE REVIEW

File: plugin.image.qwantz/addon.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.image.qwantz" name="Dinosaur Comics" version="1.0.1" provider-name="macica2">
+	<requires>
+		<import addon="xbmc.python" version="2.14.0"/>
```
Comment (enen92) [inline plugin.image.qwantz/addon.xml:4]: for leia it is advised to set version to 2.26.0

**Comment by enen92** [comment]
ping @macica2

**Comment by macica2** [comment]
Thanks @enen92, I made the update.

**Comment by enen92** [comment]
I'd be nice if you could port to python3/matrix at some point. Thanks for the submission

**Pull request merged**